dapps/onboarding.vue/src/components/sign-up/sign-up.ts

```diff
@@ -323,5 +343,78 @@ export default class SignUp extends mixins(EvanComponent) {
     // do not use $router.push to force navigation triggering!
     window.location.hash = `/${ this.$route.query.origin || getDefaultDAppEns() }`;
```

Comment:
I know it doesn't belong to the actual PR but why can't we use here the $router? Maybe $router.replace or so?

Reply:
**do not use $router.push to force navigation triggering**

Within this logic we are forcing to leave the application routing, but the vue router is started with another base routing path (.../onboarding.vue.evan). By trying to use the normal navigation at this point, it could be possible, that the onboarding will never be leaved.